hub/schema/class_label_array.py

```diff
@@ -0,0 +1,177 @@
+"""
+License:
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""
+
+from typing import List, Tuple
+from hub.schema.features import Tensor
+
+
+def _load_names_from_file(names_filepath):
+    with open(names_filepath, "r") as f:
+        return [name.strip() for name in f.read().split("\n") if name.strip()]
+
+
+class ClassLabelArray(Tensor):
```

Comment:
Overloading Array labels adds a lot of complication, I know we are trying to replace Sequence in general but need more elegant solution.